ec2/sc-ec2-linux-jumpcloud-v1.0.0.yaml

```diff
@@ -442,7 +442,7 @@ Resources:
           Value: "prod-default"
           #The session name appended to this roleid becomes the value of the AccessApprovedCaller tag
         - Key: "AccessApprovedRoleId"
-          Value: !GetAtt 'ServiceCatalogEnduser.RoleId' 
+          Value: !GetAtt 'ServiceCatalogEndusers.RoleId' 
```

Comment:
There is no `ServiceCatalogEndusers` resource in this file.  You probably want an `!ImportValue` here?

Reply:
Ah right- I revised the PR to export it from the sc-enduser-iam resources and import it here.